Mathlib/CategoryTheory/Limits/Shapes/Images.lean

```diff
@@ -250,9 +251,12 @@ variable (f)
 
 /-- Data exhibiting that a morphism `f` has an image. -/
 structure ImageFactorisation (f : X ⟶ Y) where
-  f : MonoFactorisation f
-  IsImage : IsImage F
+  /-- We can factor `f` into a composition of a monomorphism and another map -/
```

Comment:
Idem

Reply:
Added `@[inherit_doc]` to all the fields